consensus/service/src/config/tokens.rs

```diff
@@ -447,7 +574,7 @@ mod tests {
         assert_eq!(tokens, tokens2);
 
         // Validation should fail since allow_any_fee cannot be used on non-MOB tokens.
-        assert!(tokens.validate().is_err());
+        assert_validation_error(&tokens, "MOB token configuration not found");
```

Comment:
Nit: Shouldn't this be: allow_any_fee can only be used for MOB

Reply:
good catch